app/models/outcome.rb

```diff
@@ -23,4 +23,8 @@ class Outcome < ActiveRecord::Base
   def to_s
     "#{name} - #{description}"
   end
+
+  def label
+    "#{name} - #{nickname}"
```

Comment:
Going based on Mike's suggestion, we can make the name capital so that it matches how it will display in the manage outcomes views.

```
def label
    "#{name.upcase} - #{nickname}"
end 
```